tests/protractor/helper.js

```diff
@@ -95,11 +88,4 @@ module.exports = {
     var selectedElement = element(by.xpath('//*[text()[normalize-space() =  \' ' + text + '\']]'));
     return selectedElement.isPresent();
   },
-
-  takeScreenshot: function (filename) {
-    browser.takeScreenshot().then(function (png) {
-      writeScreenShot(png, filename);
```

Comment:
This feature is not active yet on Travis. It only works on local. We will need to to some tweaks to get the html reports on Travis